plugins/epos/plugin.py

```diff
@@ -1025,6 +1036,65 @@ def rda_a1_1_01d(self):
 
         return (points, msg_list)
 
+    def rda_a1_2_01d(self):
+        """Indicator RDA-A1_2-01D
+        This indicator is linked to the following principle: A1.2: The protocol allows for an
+        authentication and authorisation where necessary. More information about that principle
+        can be found here.
+        The indicator requires the way that access to the digital object can be authenticated and
+        authorised and that data accessibility is specifically described and adequately documented.
+        Technical proposal:
+
+        Returns
+        -------
+        points
+            A number between 0 and 100 to indicate how well this indicator is supported
+        msg
+            Message with the results or recommendations to improve this indicator
+        """
```

Comment:
This is the old way to use docstrings, Use instead the latest format, where it is documented how 100 points could be achieved.

Reply:
Updated

Reply:
Thanks, the description is still following the old format. Check e.g. `rda_a1_1_01d` (the one above)

Reply:
Changed